src/test/fuzz/decode_raw_psbt.cpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2009-2019 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <psbt.h>
+#include <pubkey.h>
+#include <test/fuzz/fuzz.h>
+#include <util/memory.h>
+
+void initialize()
+{
+    static const auto verify_handle = MakeUnique<ECCVerifyHandle>();
+}
+
+void test_one_input(const std::vector<uint8_t>& buffer)
+{
+    PartiallySignedTransaction decoded_psbt;
+    const std::string raw_psbt(buffer.begin(), buffer.end());
+    std::string error;
+    (void)DecodeRawPSBT(decoded_psbt, raw_psbt, error);
```

Comment:
how is this different from psbt.cpp?

Reply:
Oh, it isn't: `psbt.cpp` is a superset of this test. Removing this one :)